twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx

```diff
@@ -76,7 +76,8 @@ export default (props: Props) => {
           } else if (app.display?.messages_module?.right_icon) {
             addon_right_icon.push(app);
           } else {
-            addon_menu.push(menu_item);
+            // Not sure about this, giphy still displayed on web.qa.twake
+            app.simple_name !== 'giphy' && addon_menu.push(menu_item);
```

Comment:
@RomaricMourgues Could you check if this works on QA please? I'm not sure about this.

Reply:
No no, please revert this, removing Giphy is something we do in administration not in code :)

Reply:
What did you try to do exactly ? @stephanevieira75

Reply:
If the goal was "remove giphy" then just don't do it by code, I will handle it in the configuration of Twake directly.

Reply:
Removed 👍🏻